pyogrio/raw.py

```diff
@@ -384,12 +409,22 @@ def open_arrow(
     >>>     for table in reader:
     >>>         geometries = shapely.from_wkb(table[meta["geometry_name"]])
 
+    Or without directly returning a pyarrow object:
+
+    >>> with open_arrow(path) as source:
```

Comment:
Shouldn't this use `return_pyarrow=False`, at least to illustrate that pyarrow is not required?

```suggestion
    >>> with open_arrow(path, return_pyarrow=False) as source:
```